--- rc.vboxadd.new.orig	2009-11-01 15:23:14.000000000 +0100
+++ rc.vboxadd.new	2009-11-01 16:30:22.116899825 +0100
@@ -279,41 +279,7 @@
 
 setup()
 {
-    # don't stop the old modules here -- they might be in use
-    if find /lib/modules/`uname -r` -name "vboxvfs\.*" 2>/dev/null|grep -q vboxvfs; then
-        begin "Removing old VirtualBox vboxvfs kernel module"
-        find /lib/modules/`uname -r` -name "vboxvfs\.*" 2>/dev/null|xargs rm -f 2>/dev/null
-        succ_msg
-    fi
-    if find /lib/modules/`uname -r` -name "vboxadd\.*" 2>/dev/null|grep -q vboxadd; then
-        begin "Removing old VirtualBox vboxadd kernel module"
-        find /lib/modules/`uname -r` -name "vboxadd\.*" 2>/dev/null|xargs rm -f 2>/dev/null
-        succ_msg
-    fi
-    begin "Recompiling VirtualBox kernel modules"
-    if ! $BUILDVBOXADD \
-        --save-module-symvers /tmp/vboxadd-Module.symvers \
-        --no-print-directory install > $LOG 2>&1; then
-        fail "Look at $LOG to find out what went wrong"
-    fi
-    if [ -n "$BUILDVBOXVFS" ]; then
-        if ! $BUILDVBOXVFS \
-            --use-module-symvers /tmp/vboxadd-Module.symvers \
-            --no-print-directory install >> $LOG 2>&1; then
-            fail "Look at $LOG to find out what went wrong"
-        fi
-    fi
-    if [ -n "$BUILDVBOXVIDEO" ]; then
-        if ! $BUILDVBOXVIDEO \
-            --use-module-symvers /tmp/vboxadd-Module.symvers \
-            --no-print-directory install >> $LOG 2>&1; then
-            fail "Look at $LOG to find out what went wrong"
-        fi
-    fi
-    succ_msg
-    start
-    echo
-    echo "You should reboot your guest to make sure the new modules are actually used"
+    echo "Not implemented! Please use the virtualbox-kernel-addons.SlackBuild available at SlackBuilds.org instead."
 }
 
 dmnstatus()
